### DISCUSSION TITLE
国旗红黄我全沾，心仍向党

### DISCUSSION THREAD

**Discussion by tingsha1935** [share121/inter-knot#2276] [其他]
![3e4aa8e04db3f87c242e42a4223a116](https://github.com/user-attachments/assets/8ed28600-239d-4a5b-865e-3d0fbb94584a)

**Comment by Genshin-Neuvillette** [comment]
黄是黄，党是党。莫在本子提到党，莫将忠心染上黄

**Comment by missfggc** [comment]
什么本
